docs/events/glides/com.mbta.ctd.glides.operator_signed_in.v1.mdx

```diff
@@ -1,29 +1,63 @@
 ---
+sidebar_label: glides.operator_signed_in
 ---
 
-import GlidesExamples from "@site/examples/glides-events-schema/glides-examples.json";
 import EventSchemaPath from "@site/src/components/EventSchemaPath.tsx";
 import Examples from "@site/src/components/Examples.tsx";
+import example from "@site/examples/glides-events/operator_signed_in.v1.json";
 
-At the start of their shift, operators need to confirm that they are
-fit-for-duty and do not have any electronic devices. They currently do this by
-physically signing a paper trainsheet: in the future, they will do this
-digitally.
+# operator_signed_in
 
-Fields in the event:
+event type: `com.mbta.ctd.glides.operator_signed_in.v1`
 
-- `metadata` (Metadata): how the operator was signed in in Glides, including the inspector who signed them in.
-- `operator` (Operator): the operator who signed in
-- `signedInAt` (RFC3999 timestamp): the time at which they signed in (separate from the time of the event)
-- `signature` (Signature): how the operator confirmed that they signed in.
+At the start of their shift, operators need to confirm in Glides (on an inspector's device) that they are fit-for-duty and do not have any electronic devices.
 
-Originally described in [RFC18](https://github.com/mbta/technology-docs/blob/main/rfcs/accepted/0018-glides-trainsheet-events.md).
+## Data
+
+Fields in the event's `data`:
+
+- `metadata` ([Metadata](.#metadata)): how the operator was signed in in Glides, including the inspector who signed them in.
+- `operator` ([Operator](.#operator)): the operator who signed in
+- `signedInAt` (RFC3999 timestamp): the time at which they signed in (this is separate from the event timestamp)
+- `signature` ([Signature](#signature)): how the operator signed in.
+
+### Metadata
```

Comment:
No, nothing links here, because if it did, it should just link directly to the shared glides-events page instead.

This is here so that if someone comes here looking for information about an operator_signed_in's metadata, they can find it, instead of it being hidden on another page. They could probably find the metadata field and follow that link, but in general, the way I organized these docs was that each event page gets all the object types that show up in it, and I didn't want an object type to be missing from this event's page just because it shows up on another event so was put on the shared page instead.